.github/workflows/ci.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
       if: steps.cache-dev-deps.outputs.cache-hit != 'true'
       run: |
         python -m pip install --upgrade pip
-        pip install -r dev-requirements.txt
+        pip install -r dockerfiles/dev-requirements.txt
```

Comment:
@Hedingber i wouldn't change the location of `requirements.txt` to the docker subdir, remember its also used by PIP and people would expect to see the requirements near the setup.py

Reply:
The way I see it is to have a `requirements` dir where you can put those files. since it is a manner of flavor, you decide

Reply:
Going with @yaronha 's suggestion for now - moving `requirements.txt` and `dev-requirements.txt` to root dir